### ISSUE TITLE
[BUG] <title> Is there a way to output logits?

### ISSUE THREAD

**Issue by zhaowenZhou** [OpenBMB/MiniCPM-o#653]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

I've tried plenty of ways to let model output its logits, including  return_dict_in_generate=True, output_scores=True, but I still cannot get the logits

### 复现方法 | Steps To Reproduce

        generated_dict = self.model.chat(
            image=None,
            mgs=model_inputs,
            tokenizer=self.tokenizer,
            max_new_tokens=512,
            return_dict_in_generate=True,
            output_scores=True
        )

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by LDLINGLINGLING** [comment]
你好，请看minicpm-cookbook的仓库，其中有如何过得logits的教程和代码
